packages/components/src/SignConstitutionButton.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+import { SignConstitutionButton } from "./SignConstitutionButton";
+import { Civil } from "@joincivil/core";
+
+let civil: Civil | undefined;
+
+try {
+  civil = new Civil();
+} catch (error) {
+  civil = undefined;
+}
+
+const signConstitution = async (): Promise<void> => {
+  console.log("Signing the Constituion!");
+};
+
+const Wrapper = styled.div`
+  margin: 50px;
+  max-width: 500px;
+`;
+
+storiesOf("Sign Constitution Button", module)
+  .add("User can sign", () => {
+    return (
+      <Wrapper>
+        <SignConstitutionButton
+          civil={civil}
+          requiredNetwork="rinkeby"
```

Comment:
I'm not sure but the network returned might a number (4 for rinkeby) not the name

Reply:
I added a method to get it by name